pkgs/applications/misc/yubioath-desktop/default.nix

```diff
@@ -48,11 +48,18 @@ mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    description = "Yubikey Desktop Authenticator";
+    description = "Yubico Authenticator";
+    long-description = ''
+      Application for generating Open Authentication (OATH) time-based TOTP and
+      event-based HOTP one-time password codes, with the help of a YubiKey that
+      protects the shared secrets.
+    '';
 
-    homepage = https://www.yubico.com/support/knowledge-base/categories/articles/yubico-authenticator-download/;
+    homepage = https://developers.yubico.com/yubioath-desktop;
+    downloadPage = https://developers.yubico.com/yubioath-desktop/Releases/;
+    changelog = https://developers.yubico.com/yubioath-desktop/Release_Notes.html;
```

Comment:
URLs should be quoted. See RFC 45 for more information: https://github.com/NixOS/rfcs/pull/45.

Reply:
Fixed. I did not comment as it was not previously. Will keep that in mind!